<launch>
  <arg name="sim_mode" default="false"/>
  <param name="/use_sim_time" value="$(arg sim_mode)"/>
  <arg name="loopTimes" default="0"/>
  <node pkg="kairobot" type="multi_goals.py" respawn="false" name="multi_goals" output="screen">
    <param name="goalListX" value="[7.583, 9.543, 9.543, 10.648, 10.854, 12.941, 0.0]"/>
    <param name="goalListY" value="[0.117, 0.308, 0.308, 6.229, 14.035, 16.133, 0.0]"/>
    <param name="goalListZ" value="[0.019, 0.646, 0.649, 0.643, 0.616, 0.026, 0.0]"/>
    <param name="loopTimes" value="1"/>
    <param name="map_frame" value="map"/>
  </node>
</launch>
